ouroboros-network/src/Ouroboros/Network/PeerSelection/RootPeersDNS.hs

```diff
@@ -70,16 +72,24 @@ import           Ouroboros.Network.PeerSelection.RootPeersDNS.DNSActions
 --
 
 data TraceLocalRootPeers peerAddr exception =
-       TraceLocalRootDomains [(Int, Map RelayAccessPoint PeerAdvertise)]
+       TraceLocalRootDomains [( HotValency
+                              , WarmValency
+                              , Map RelayAccessPoint PeerAdvertise)]
        -- ^ 'Int' is the configured valency for the local producer groups
      | TraceLocalRootWaiting DomainAccessPoint DiffTime
      | TraceLocalRootResult  DomainAccessPoint [(IP, DNS.TTL)]
-     | TraceLocalRootGroups  [(Int, Map peerAddr PeerAdvertise)]
+     | TraceLocalRootGroups  [( HotValency
+                              , WarmValency
+                              , Map peerAddr PeerAdvertise)]
        -- ^ This traces the results of the local root peer provider
      | TraceLocalRootDNSMap  (Map DomainAccessPoint [peerAddr])
        -- ^ This traces the results of the domain name resolution
-     | TraceLocalRootReconfigured [(Int, Map RelayAccessPoint PeerAdvertise)] -- ^ Old value
-                                  [(Int, Map RelayAccessPoint PeerAdvertise)] -- ^ New value
+     | TraceLocalRootReconfigured [( HotValency
+                                   , WarmValency
+                                   , Map RelayAccessPoint PeerAdvertise)] -- ^ Old value
+                                  [( HotValency
+                                   , WarmValency
+                                   , Map RelayAccessPoint PeerAdvertise)] -- ^ New value
```

Comment:
I wonder how useful it is to trace the old value. I've been reading the logs recently and I was mostly ignoring the previous value.  It's in the logs already but further back in the log.

Reply:
I like it if one is parsing the logs and wants to run some diff, it should be easier this way. Is not like logs should have *less* information if possible